### PR TITLE
docs: Document Pager InputType and PageSizes

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: components/pager/events.md [hunk 1]
```diff
@@ -36,19 +35,52 @@ The `PageChanged` event fires a new page is selected. You can use it to implemen
...
+        CurrentPage = newPage;
+        Result = $"Current page: {newPage}";
     }
 }
 ````
 >caption The result from the code snippet above
 
 ![config of the pager with one-way binding](images/pager-data-binding.gif)
 
+## PageSizeChanged
```
Comment (marin-bratanov) [inline components/pager/events.md:51]: Did the grid also get a PageSizeChanged event?

Also, I recommend adding a list of the available events at the beginning of the article to ensure people see what is available.

File: components/pager/overview.md [hunk 2]
```diff
@@ -174,6 +180,8 @@ You can avoid loading all the data at once, as this can be a costly operation. I
 
 ### Two-way Binding
 
+This section shows how to manually implement a behavior that is **built-in** as of version 2.26. See [`InputType`](#features) above.
```
Comment (marin-bratanov) [inline components/pager/overview.md:183]: This section shows how to two-way bind the `Page` not `PageSize` parameter. This sentence either needs more explanations, or a separate code example.

**Review by marin-bratanov — CHANGES_REQUESTED** [review]
I expect docs for the grid features be done separately?

File: components/pager/events.md (hunk 1 above)
Comment (dimodi) [reply components/pager/events.md:51]: Yes, the Grid and ListView-related task is separate.

Will add an event list on top.

File: components/pager/overview.md (hunk 2 above)
Comment (dimodi) [reply components/pager/overview.md:183]: My mistake, thanks.

**Review by marin-bratanov — APPROVED** [review]

**Pull request merged**